lib/membrane/rtp/twcc_sender.ex

```diff
@@ -123,10 +123,17 @@ defmodule Membrane.RTP.TWCCSender do
     buffer =
       Header.Extension.put(buffer, %Header.Extension{identifier: :twcc, data: <<seq_num::16>>})
 
+    payload_size =
+      if Map.get(buffer.metadata.rtp, :is_padding?, false) do
+        8 * 255
+      else
+        bit_size(buffer.payload)
+      end
```

Comment:
What is acctually in `buffer.payload` here? If the whole serialized RTP packet then `bit_size(buffer.payload)` should work for padding packets too

Reply:
I'm pretty sure this is before the serializer, so payload for padding packets is empty. Actually, now that you pointed it out, shouldn't we count the header as well!? I guess for most packet it will be inconsequential, but leaving it as it is seems a little... wrong.

Reply:
I am not sure if we can do anything with this. I would leave `FIXME` tag with the explanation. The whole code will work more consistently anyway